src/DIRAC/Core/Utilities/DIRACScript.py

```diff
@@ -0,0 +1,126 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+__RCSID__ = "$Id$"
+
+from collections import defaultdict
+import functools
+
+import six
+
+from DIRAC import gLogger
```

Comment:
I'll just get rid of the logging, it could be useful for understanding where a script is coming from but you definitely don't want to see it all the time and it will be hard actually get the verbose messages out (as `-ddd` won't have been parsed yet).